src/lib/snack-bar/snack-bar-ref.ts

```diff
@@ -38,8 +43,21 @@ export class MdSnackBarRef<T> {
     }
   }
 
+  /** Marks the snackbar action clicked. */
+  action(): void {
```

Comment:
Can we mark this as internal (just an underscore prefix without `private` or `@internal`)? I don't think users should be calling it on their own.